python/lsst/ap/verify/pipeline_driver.py

```diff
@@ -65,8 +65,9 @@ def runApPipe(workspace, parsedCmdLine):
 
     Returns
     -------
-    dataIds : `lsst.pipe.base.DataIdContainer`
-        The set of complete data IDs fed into ``ap_pipe``.
+    result : `Struct`
+        The `Struct` returned from `ApPipeTask.parseAndRun` with
+        ``doReturnResults=False``.
```

Comment:
This is the part where I express dismay and/or confusion that we return `result` which explicitly contains something that says `doReturnResults=False`. Can you give me a little more context for the confusing terminology here?

Reply:
`doReturnResults` is a [keyword argument to `parseAndRun`](https://pipelines.lsst.io/py-api/lsst.pipe.base.CmdLineTask.html#lsst.pipe.base.CmdLineTask.parseAndRun). I think the logic is that `parseAndRun` returns things that are not results? 🤷‍♂